test/AnchorRepository.test.js

```diff
@@ -22,19 +18,6 @@ function createSignatureMessage(payloads) {
     return bufferToHex(keccak(Buffer.concat(buffers)));
 }
 
-function getDeterministCommitParameter(accounts) {
```

Comment:
I think I added this test case to easier compare the node implementation with smart contracts tests.

Reply:
If we don't need them anymore it's fine for me

Reply:
Well we need to generate new values with the new centrifuge Id. Seemed hard to do that. We have good enough tests and we will remove that logic in the future so just removing them made sense